src/Rule/Nested.php

```diff
@@ -80,6 +85,7 @@ public function __construct(
         private bool $requirePropertyPath = false,
         private string $noPropertyPathMessage = 'Property path "{path}" is not found.',
         private bool $normalizeRules = true,
+        private bool $propagateOptions = false,
```

Comment:
Is there a reason it's `false` by default? Are there valid cases when we want it `false`?

Reply:
To configure other `Nested` rules within base `Nested` rule separately like these ones:

- https://github.com/yiisoft/validator/blob/5d1a1f48422cf44bd050564086ad79a7aa167695/tests/Rule/NestedHandlerTest.php#L505
- https://github.com/yiisoft/validator/blob/5d1a1f48422cf44bd050564086ad79a7aa167695/tests/Rule/NestedHandlerTest.php#L508

Also otherwise in the same example options [here](https://github.com/yiisoft/validator/blob/5d1a1f48422cf44bd050564086ad79a7aa167695/tests/Rule/NestedHandlerTest.php#L508) will be set 3 times.